databricks_cli/pipelines/cli.py

```diff
@@ -49,28 +50,168 @@
 
 
 @click.command(context_settings=CONTEXT_SETTINGS,
-               short_help='Deploys a pipeline according to the pipeline specification.')
-@click.argument('spec_arg', default=None, required=False)
-@click.option('--spec', default=None, type=PipelineSpecClickType(), help=PipelineSpecClickType.help)
+               short_help='Creates a pipeline.')
+@click.argument('settings_arg', default=None, required=False)
+@click.option('--settings', default=None,
+              type=PipelineSettingClickType(), help=PipelineSettingClickType.help)
 @click.option('--allow-duplicate-names', is_flag=True,
-              help="If true, skips duplicate name checking while deploying the pipeline.")
+              help="If true, skips duplicate name checking while creating the pipeline.")
+@debug_option
+@profile_option
+@pipelines_exception_eater
+@provide_api_client
+def create_cli(api_client, settings_arg, settings, allow_duplicate_names):
+    # pylint: disable=line-too-long
+    """
+    Creates a pipeline according to the pipeline settings. The pipeline settings are a
```

Comment:
How about "Creates a pipeline specified by the pipeline settings"?

Reply:
Yeah I like that, updated it thanks